### ISSUE TITLE
Broken message_id generation

### ISSUE THREAD

**Issue by nicholaszuccarelli** [juhasev/laravel-ses#24]
Copying details from https://github.com/juhasev/laravel-ses/pull/20 comments

The changes in this commit have broken message_id usage.
The laravel_ses_sent_emails table is now storing the message_id as: (Notice the "Message-ID" portion in the string.
![image](https://github.com/juhasev/laravel-ses/assets/13761766/7734ad6c-2f8c-455e-b35a-81b84c715fd2)

This commit also doesn't fix the issue I was attempting to resolve by fetching the generated Message-ID at time of email execution.
With v2 of the package, I could do (within a Laravel Mailable)
![image](https://github.com/juhasev/laravel-ses/assets/13761766/0fdb2124-9469-42a9-ad92-73491c444cf9)
Which basically let me fetch that exact message-ID generated _before_ the email was sent, however with v3, withSymfonyMessage no longer contains the Message-ID header.

As far as I can tell, this package doesn't offer a way to return the LaravelSesSentEmail model directly.

**Comment by leMaur** [comment]
Hi @nicholaszuccarelli,
thanks for your troubleshooting!

I've released the version 3.3.0 that fixes the issue with the Message-ID generation.

About your second snippet, where you try to get the Message-ID from the header, I suggest you to use the `\Juhasev\LaravelSes\ModelResolver::class`.

I've also added a new method `getMessageId` in the contract so you can use it like

```php
use Juhasev\LaravelSes\ModelResolver;

$sentEmail = ModelResolver::get('SentEmail')::whereEmail('john.doe@example.com')->first();

$messageId = $sentEmail->getMessageId();

```

Anyway, you should be able get the Message-ID from the header.
Keep in mind that now we set this header in the `appendToHeaders` method after calling `initMessage`.

You can see the implementation here 👉 https://github.com/juhasev/laravel-ses/blob/master/src/SesMailer.php#L208

**Comment by nicholaszuccarelli** [comment]
Thank you. I'll have a look soon and see if these new changes work in my situation.
